File: app/src/main/java/com/admin/theapp/HotelsApp.java
package com.admin.theapp;

import android.support.v7.app.AppCompatDelegate;

import dagger.android.AndroidInjector;
import dagger.android.DaggerApplication;

public class HotelsApp extends DaggerApplication {

    @Override
    public void onCreate() {
        super.onCreate();
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
    }

    @Override
    protected AndroidInjector<? extends DaggerApplication> applicationInjector() {
        return DaggerAppComponent.builder().create(this);
    }
}

File: app/src/main/java/com/admin/theapp/HotelsApp.kt
package com.admin.theapp

import android.support.v7.app.AppCompatDelegate
import com.admin.theapp.dagger.DaggerAppComponent
import dagger.android.AndroidInjector
import dagger.android.DaggerApplication

class HotelsApp : DaggerApplication() {

    override fun onCreate() {
        super.onCreate()
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
    }

    override fun applicationInjector(): AndroidInjector<out DaggerApplication> {
        return DaggerAppComponent.builder().create(this)
    }
}
